molgenis-data/src/main/java/org/molgenis/data/decorator/example/SpyingRepositoryDecorator.java

```diff
@@ -0,0 +1,30 @@
+package org.molgenis.data.decorator.example;
+
+import org.molgenis.data.AbstractRepositoryDecorator;
+import org.molgenis.data.Entity;
+import org.molgenis.data.Repository;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class SpyingRepositoryDecorator extends AbstractRepositoryDecorator<Entity>
```

Comment:
Move examples to test folder?

Reply:
Will be removed or moved to integration test suite.

Reply:
When moved to the test folder we do not have a way to demo and test this since none of the planned dynamic decorators are build yet

Reply:
Demo by executing a test or locally with a dummy dynamic decorator.